packages/std-client/src/setup/setupMUDNetwork.ts

```diff
@@ -65,7 +68,25 @@ export async function setupMUDNetwork<C extends ContractComponents, SystemTypes
     )
   );
 
+  const storeSchemaTableId = new TableId("mudstore", "schema");
+  const storeSchemaComponent = defineComponent(
+    world,
+    { schema: Type.T },
```

Comment:
Note: with latest `main` we probably need to add `keyType` here too (and when decoding schemas), see #480